boards/pjrc/teensy4/board.yml

```diff
@@ -9,3 +9,8 @@ boards:
     vendor: pjrc
     socs:
       - name: mimxrt1062
+  - name: teensyMM
```

Comment:
```suggestion
  - name: teensymm
```

Reply:
Made changes, still working on all of the suggestions and then will squash all of the commits again